.github/workflows/pylint.yml

```diff
@@ -0,0 +1,45 @@
+
+
+name: Pylint
+
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8  # maybe better not hardcoded
```

Comment:
You should not be so sensible to the Python version, so whatever is the default should be fine.

If instead you really want to stick to py3.8, because otherwise Pylint will enable some different lints (I don't think it would, but it's possible, I should check), then it is fine to hard-code, it doesn't make a big difference (in principle you could define a variable top-level, but it is even fine like this, do not overcomplicate things).